bin/latex.py

```diff
@@ -370,10 +374,9 @@ def build_problem_pdf(
     Arguments:
     -- language: str, the two-letter language code appearing the file name, such as problem.en.tex
     """
-    main_file = build_type.value
-    main_file += "-web.tex" if web else ".tex"
+    main_file = build_type.path(ext="-web.tex" if web else ".tex").name
 
-    bar = PrintBar(f"{main_file[:-3]}{language}.pdf")
+    bar = PrintBar(f"{main_file[:4]}.{language}.pdf")
```

Comment:
This now renders as `prob.en.pdf` or `solu.en.pdf`, that's probably not intended? :stuck_out_tongue: